settings.py

```diff
@@ -83,7 +83,7 @@
         'content_host': 'https://stable.dev.lcip.org',
         'oauth_host': 'https://oauth-stable.dev.lcip.org/v1',
         'profile_host': 'https://stable.dev.lcip.org/profile/v1',
-        'redirect_url': 'http://localhost:3000/fxa-authenticate',
+        'redirect_url': 'http://olympia.test/api/v3/accounts/authenticate/',
```

Comment:
As I recall, `redirect_url` has no effect whatsoever. The value configured at https://oauth-stable.dev.lcip.org/console/ will override it.

Reply:
We like to keep them in sync for reference for reference though

Reply:
yep, but I prefer to use the values that should be there. That will avoid confusion in the future I believe. On a related note, FxA should really take `redirect_url` into account (and check its value against the redirect URI it knows).

Reply:
You shouldn't change this without changing https://oauth-stable.dev.lcip.org/console/client/0f95f6474c24c1dc or replacing it with a new oauth client id/secret pair.

Reply:
> That will avoid confusion in the future I believe.

It's still pretty confusing 🤣 but, yeah, whatever works. I think we should delete them but maybe something will break, who knows.